.github/workflows/clippy_bors.yml

```diff
@@ -77,6 +77,7 @@ jobs:
       run: |
         sudo dpkg --add-architecture i386
         sudo apt-get update
+        sudo apt-get -y upgrade
```

Comment:
This costs ~2 min of CI time. I think this is ok, since the integration tests take about 16 minutes and this check with this change ~12min.